### ISSUE TITLE
session.followRefer() fails silently unless mediaHandlerFactory is redefined

### ISSUE THREAD

**Issue by growtofill** [onsip/SIP.js#300]
The simplest example (taken from the docs) fails for me (if there was a hold before REFER, the client stays forever on the remote hold):

```
session.on('refer', session.followRefer(
    () => console.debug('Got new session')
));
```

I've found out why it happens:
- because [`onIceCompleted` promise](https://github.com/onsip/SIP.js/blob/0.7.3/src/WebRTC/MediaHandler.js#L509) never resolves;
- because ICE never starts;
- because no stream has been added to the peer connection;
- because [`MediaHandler#getReferMedia`](https://github.com/onsip/SIP.js/blob/0.7.3/src/WebRTC/MediaHandler.js#L161-L178) returns `{constraints: {audio:false, video:false}}`;
- because remote stream has neither audio, nor video tracks.

Firstly, is it ok for the remote stream to not have tracks? I guess it is, because if I redefine a `getReferMedia` method referring works:

```
mediaHandlerFactory: (session, sessionOptions) => (
    Object.assign(
        WebRTC.MediaHandler.defaultFactory(session, sessionOptions),
        { getReferMedia: () => options.media }
    )
)
```

Secondly, in any case, it should not fail silently.

**Comment by egreenmachine** [comment]
We believe that this issue may be resolved in the latest tagged version [0.8.0](https://github.com/onsip/SIP.js/releases/tag/0.8.0). Can you please retest with that version and let us know if this is still an issue. We will close this issue on 11/1/17 if we do not hear back. Thanks.

**Comment by egreenmachine** [comment]
Can you try out #459? I think that I have addressed this issue there. 

* Due to the age of this issue, if we do not hear any feedback on this we will close it once we merge the pull request.
